lib/c-api/src/wasm_c_api/unstable/parser/operator.rs

```diff
@@ -1,4 +1,4 @@
-use wasmer_api::wasmparser::Operator;
+use wasmparser::Operator;
```

Comment:
Why this change? `wasmparser` is not a dependency of `wasmer-c-api`.

Reply:
I've reverted this but I did have to make change in `lib/api/src/sys/mod.rs` so `wasmer_api::wasmparser` is available when the `compiler` feature isn't enabled.